src/slowmokit/methods/activation_functions.cpp

```diff
@@ -0,0 +1,83 @@
+/**
+ * @file methods/activation_functions.cpp
+ *
+ * Implementation of activation functions
+ */
+#include "activation_functions.hpp"
+template<class T>
+// sigmoid
+double sigmoid(double x)
```

Comment:
You need to write these functions for a vector and not a single value

Reply:
> You need to write these functions for a vector and not a single value

I'm changing all the functions

Reply:
> You need to write these functions for a vector and not a single value

all changes updated